controllers/solrprometheusexporter_controller.go

```diff
@@ -197,28 +197,30 @@ func (r *SolrPrometheusExporterReconciler) Reconcile(req ctrl.Request) (ctrl.Res
 					prometheusExporter.Spec.SolrReference.SolrTLS.KeyStorePasswordSecret.Key, keyStorePasswordSecret.Name)
 			}
 
-			tlsClientOptions = &util.TLSClientOptions{}
```

Comment:
The `TLSClientOptions` name was misleading once I repurposed this struct to be used by the exporter and the StatefulSet code